docs/index.md

```diff
@@ -0,0 +1,43 @@
+# Tmate-ssh-server Operator
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+deploying and managing [Tmate self-hosted server](https://tmate.io/). Tmate is an
+open source terminal multiplexer, providing instant terminal sharing capabilities.
+
+Tmate is a terminal multiplexer that allows remote terminal sharing. It enables users to share
```

Comment:
This seems a bit like the opening paragraph?

Reply:
Removed the opening sentence, it now reads as `Tmate enables users to share ...`